test/fixtures/minio-fixture/build.gradle

```diff
@@ -6,6 +6,26 @@
  * Side Public License, v 1.
  */
 apply plugin: 'elasticsearch.test.fixtures'
+apply plugin: 'java'
+apply plugin: 'elasticsearch.java'
 
 description = 'Fixture for MinIO Storage service'
 
+configurations.all {
+  transitive = false
+}
+
+dependencies {
+  testImplementation project(':test:framework')
+
+  api "junit:junit:${versions.junit}"
+  implementation "com.carrotsearch.randomizedtesting:randomizedtesting-runner:${versions.randomizedrunner}"
+  implementation 'org.testcontainers:testcontainers:1.19.2' //replace with the latest version
```

Comment:
Should we add this to `versions.properties`?

Reply:
done